pkg/node/peer_manager/peer_manager.go

```diff
@@ -109,27 +109,28 @@ func (a *PeerManagerImpl) NewConnection(p peer.Peer) error {
 
 	if p.Handshake().Version.CmpMinor(a.version) >= 2 {
 		err := errors.Errorf(
-			"versions are too different, current %s, connected %s",
+			"versions are too different, current %s, connected %s (peer '%s')",
 			a.version.String(),
 			p.Handshake().Version.String(),
+			p.ID(),
 		)
 		a.restrict(p, now, err.Error())
 		_ = p.Close()
-		return proto.NewInfoMsg(err)
```

Comment:
Why move this error logging from DEBUG to ERROR level?

Reply:
That shouldn't be there. I forgot to return it back. Fixed.